api/routes/family.py

```diff
@@ -85,23 +94,35 @@ async def get_pedigree(
         replace_with_participant_external_ids=replace_with_participant_external_ids,
         replace_with_family_external_ids=replace_with_family_external_ids,
         empty_participant_value=empty_participant_value,
-        include_header=include_header,
+        include_header=True,
     )
 
-    output = io.StringIO()
-    writer = csv.writer(output, delimiter='\t')
-    writer.writerows(pedigree_rows)
+    if response_type == ContentType.CSV or response_type == ContentType.TSV:
+        delim = '\t' if response_type == ContentType.TSV else ','
+        output = io.StringIO()
+        writer = csv.writer(output, delimiter=delim)
 
-    basefn = f'{connection.project}-{date.today().isoformat()}'
+        if not include_header:
+            next(pedigree_rows)
 
-    if internal_family_ids:
-        basefn += '-'.join(str(fm) for fm in internal_family_ids)
+        writer.writerows(pedigree_rows)
 
-    return StreamingResponse(
-        iter(output.getvalue()),
-        media_type='text/csv',
-        headers={'Content-Disposition': f'filename={basefn}.ped'},
-    )
+        basefn = f'{connection.project}-{date.today().isoformat()}'
+
+        if internal_family_ids:
+            basefn += '-'.join(str(fm) for fm in internal_family_ids)
+
+        return StreamingResponse(
+            iter(output.getvalue()),
+            media_type=f'text/{response_type}',
+            headers={'Content-Disposition': f'filename={basefn}.ped'},
+        )
+    else:
+        header = pedigree_rows.pop(0)
+        header = [re.sub(r'^[^a-zA-Z0-9]+', '', x) for x in header]
```

Comment:
This is great! On reflection, could we convert anything to camelcase, eg so: `"Participant ID"` would go to `participant_id`, I imagine something like `.lower().replace(' ', '_')` before the `re.sub` would be perfect.

Reply:
Done 😉